Modules/Loadable/Markups/qSlicerMarkupsModuleWidget.cxx

```diff
@@ -1778,7 +1780,7 @@ void qSlicerMarkupsModuleWidget::onRestoreControlPointPushButtonClicked()
     }
 }
 //-----------------------------------------------------------------------------
-void qSlicerMarkupsModuleWidget::onUnsetControlPointPushButtonClicked()
+void qSlicerMarkupsModuleWidget::onUnsetControlPointPushButtonClicked()  /// HERE
```

Comment:
```suggestion
void qSlicerMarkupsModuleWidget::onUnsetControlPointPushButtonClicked()
```

Reply:
Thanks for cleaning up my mental note comment. I forgot I left that there